data_steward/tools/import_rdr_omop.sh

```diff
@@ -57,12 +62,12 @@ export GOOGLE_APPLICATION_CREDENTIALS="${KEY_FILE}"
 export GOOGLE_CLOUD_PROJECT="${app_id}"
 export BIGQUERY_DATASET_ID="${OUTPUT_DATASET}"
 
-today=$(date '+%Y%m%d')
-RDR_DATASET="${today}_rdr"
+
+RDR_DATASET="rdr${RDR_UPLOAD_DATE}"
 
 source "${TOOLS_DIR}/set_path.sh"
 
-bq mk -f --description "RDR DUMP loaded from ${RDR_DIRECTORY} on ${today}" "${GOOGLE_CLOUD_PROJECT}:${RDR_DATASET}"
+bq mk -f --description "RDR DUMP loaded from ${RDR_DIRECTORY} on ${RDR_UPLOAD_DATE}" "${GOOGLE_CLOUD_PROJECT}:${RDR_DATASET}"
```

Comment:
nit: Change to `dated` instead of `on` since it can be confusing.
```suggestion
bq mk -f --description "RDR DUMP loaded from ${RDR_DIRECTORY} dated ${RDR_UPLOAD_DATE}" "${GOOGLE_CLOUD_PROJECT}:${RDR_DATASET}"
```